docs/supported.md

```diff
@@ -53,14 +53,14 @@ See also [letters](#letters)
 
 ||||||
 |:-----------------------------------|:---------------------------------------|:----------|:-------------------------------------------------------|:-----
-|$( )$ `( )` |$\lt~\gt$ `\lt \gt` |$⌈~⌉$ `⌈ ⌉`|$\lceil~\rceil$ `\lceil`<br>$~~~~~$`\rceil`  |$\uparrow$ `\uparrow`
+|$( )$ `( )` |$\lparen~\rparen$ `\lparen`<br>$~~~~$`\rparen`|$⌈~⌉$ `⌈ ⌉`|$\lceil~\rceil$ `\lceil`<br>$~~~~~$`\rceil`  |$\uparrow$ `\uparrow`
 |$[ ]$ `[ ]` |$\lbrack~\rbrack$ `\lbrack`<br>$~~~~$`\rbrack`|$⌊~⌋$ `⌊ ⌋`|$\lfloor~\rfloor$ `\lfloor`<br>$~~~~~$`\rfloor` |$\downarrow$ `\downarrow`
 |$\{ \}$ `\{ \}`|$\lbrace \rbrace$ `\lbrace`<br>$~~~~$`\rbrace`|$⎰⎱$ `⎰⎱`  |$\lmoustache \rmoustache$ `\lmoustache`<br>$~~~~$`\rmoustache`|$\updownarrow$ `\updownarrow`
```

Comment:
I noticed an inconsistency here in whether there is a `~` between two delimiters. In my opinion, we should add a `~` in `$( )$` and `[ ]` so that they match the longer forms `$\lparen~\rparen$` and `$\lbrack~\rbrack$`. I also find it weird that the source columns do not show the `~` in the source. @ronkok, any comments or suggestions?

Reply:
> I do not see any examples of that, at least not in the delimiter table.

They are all like that. ```$\lt~\gt$ `\lt \gt` ``` is perhaps most obvious; I'm suggesting it should be ```$\lt~\gt$ `\lt~\gt` ```.  But also e.g. ``` `\lparen`<br>$~~~~$`\rparen` ``` should be ``` `\lparen`<br>$~~~~$`~\rparen` ``` or similar.  Should we do this in a separate PR?

Reply:
Hmm, maybe those aren't meant to be source columns, just lists of command sequences?  Then perhaps ``` `\lt \gt` ``` should be ``` `\lt` `\gt` ``` (a less important change).

Reply:
Ah, now I understand the question. It was a presentation decision. In the KaTeX renderings, I inserted `~` spaces to make the delimiters visually distinct from each other. If I needed multiple `~` spaces, I used them. 

To the right of each rendering, I wrote the code that would produce the delimiters. I wasn't trying to be strictly precise on space handling. I was just trying to present the information that an author would need to write delimiters.

And by that crierion, a `~` would help in the `$( )$` and `$[ ]$` locations.